packages/next/build/webpack/loaders/next-serverless-loader/utils.ts

```diff
@@ -109,13 +107,9 @@ export function getUtils({
     }
 
     const checkRewrite = (rewrite: Rewrite): boolean => {
-      const matcher = getPathMatch(
-        rewrite.source + (trailingSlash ? '(/)?' : ''),
-        {
-          removeUnnamedParams: true,
-          strict: true,
-        }
-      )
+      const matcher = getPathMatch(removeTrailingSlash(rewrite.source), {
```

Comment:
I updated this because it was the same change from the same original PR. But I don't think it was breaking anything because I didn't see any cases where `trailingSlash` was passed into `getUtils`.